main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -444,6 +444,8 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
   def reconfigureFlix(flix: Flix): Unit = {
     val previousSources = timestamps.keySet
 
+    implicit val defaultSctx: SecurityContext = SecurityContext.AllPermissions
```

Comment:
@AStenbaek Here we will need to discover the right context based on the manifests.